### DISCUSSION TITLE
react/useEffect-and-useLayoutEffect

### DISCUSSION THREAD

**Discussion by giscus** [howdy-mj/blog-v2#53] [Announcements]
# react/useEffect-and-useLayoutEffect

예전에 데이터에 따라 DOM 조작이 필요했었는데, 이때 화면의 깜빡임을 없애기 위해 useEffect 대신 useLayoutEffect를 사용했었다.

그러나 정확히 이 둘의 차이가 무엇인지 생각해보면, 대답할 수 없었다.

## useEffect

React의 re...

https://www.howdy-mj.me/react/useEffect-and-useLayoutEffect

**Comment by skh417** [comment]
좋아요

**Comment by youngrongoh** [comment]
리액트 내부에서 바이너리를 이용해 플래그 관리를 하고 있었던 점이 흥미롭네요. 리액트 내부 코드까지 찾아가며 학습하시는 모습에 자극받고 갑니다. 좋은 콘텐츠 감사합니다.🙇‍♂️
